.github/workflows/tests.yml

```diff
@@ -78,6 +78,28 @@ jobs:
         run: tox -e py
       - name: Run slow tests
         run: tox -e integration
+  windows:
+    if: "contains(github.event.head_commit.message, 'Trigger CI')"
+    name: Windows
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [ 3.7, 3.8 ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
```

Comment:
@cthoyt Is there a reason you install a new conda instance and aren't using the default one that comes with the Github Actions virtual environment pre-installed?

Reply:
https://github.com/conda-incubator/setup-miniconda#conda-incubatorsetup-miniconda Apparently they try to locate the pre-installed ones. So this shouldn't be the reason for the slowness 🤷‍♂️